I can't execute some commands

A little background)
I have updated Windows, respectively, all related programs and software. Up to this point, everything worked.

What is happening now is that manually calling any of my project's commands (and those written by webnative) from the terminal is working, but if I go to webnative and click sync, the output tab displays this
[wn] Syncing...
> pnpx cap sync
 ERR_PNPM_DLX_NO_BIN  No binaries found in cap

[error] pnpx cap sync Failed
[error] Sync Failed.

I really tried to Google it, but I didn't find a solution in my case. I even tried setting the alias manually as `pnpx = pnpm'...


The questions include the following:

1. What is pnpx and why is the extension trying to work through it?

2. Why do the settings of the "Webnative: Package Manager" extension not affect the commands being called?

P.S.
I use pnpm as the main package manager.
pnpm v10.20
node v24
I also tried reinstalling the extension.

--- a/CHANGELOG.md
+++ b/CHANGELOG.md
@@ -1,5 +1,9 @@
 ## Changelog
 
+### Version 2.0.53
+
+- Fix pnpm project to use pnpm exec rather than pnpx
+
 ### Version 2.0.52
 
 - Fix live reload on Windows by using start /B for parallel process execution

--- a/package.json
+++ b/package.json
@@ -2,7 +2,7 @@
   "name": "webnative",
   "displayName": "WebNative",
   "description": "Create and maintain web and native projects",
-  "version": "2.0.52",
+  "version": "2.0.53",
   "icon": "media/webnative.png",
   "publisher": "WebNative",
   "keywords": [

--- a/src/node-commands.ts
+++ b/src/node-commands.ts
@@ -318,7 +318,7 @@ export function npx(project: Project, options?: NpxOptions): string {
     case PackageManager.bun:
       return `${InternalCommand.cwd}bunx`;
     case PackageManager.pnpm:
-      return options?.forceNpx ? `${InternalCommand.cwd}npx` : `${InternalCommand.cwd}pnpx`;
+      return options?.forceNpx ? `${InternalCommand.cwd}npx` : `${InternalCommand.cwd}pnpm exec`;
     case PackageManager.yarn:
       if (options?.forceNpx && !project.isModernYarn()) {
         return `${InternalCommand.cwd}npx`;

